### DISCUSSION TITLE
direct filteren by URL

### DISCUSSION THREAD

**Discussion by reiniertc** [jonaswinkler/paperless-ng#1524] [Support]
Is it possible to directly open paperless-ng via an url, with certain selections prefilled?
like for example:
http:<<url to paperless>>/docmuents/?tag=todo
or
http:<<url to paperless>>/view/3

I've tried several options, can't find any clue in the docs.

**Comment by RefineryX** [comment]
I had this question too. Trying to do the same via ASN
